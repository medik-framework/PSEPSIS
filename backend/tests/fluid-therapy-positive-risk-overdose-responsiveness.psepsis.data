{
  "fluidOverdoseRisks": null,
  "fluidOverdoseSigns": null,
  "normalSaline": 20
}
