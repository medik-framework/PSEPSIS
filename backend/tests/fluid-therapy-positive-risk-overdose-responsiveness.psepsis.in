[
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "StartFluidTherapy",
    "eventArgs": []
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "FluidOverloadRisksEntered",
    "eventArgs": [1],
    "waitFor" : {
      "name": "Instruct",
      "args": ["get fluid overload risks"]
    },
    "updateData": {
      "name": "fluidOverloadRisks",
      "args": 0
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "ConfirmFluidAdministered",
    "eventArgs": [],
    "waitFor" : {
      "name": "RecommendDrug",
      "args": [
        "Normal Saline",
        "Fluid Therapy",
        "5",
        "administer 5-10 ml/kg Normal Saline"
      ]
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "FluidResponsivenessEntered",
    "eventArgs": [1],
    "waitFor" : {
      "name": "Instruct",
      "args": ["get responsiveness to fluids"]
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "ConfirmFluidOverloadSignsEntered",
    "eventArgs": [1],
    "waitFor" : {
      "name": "Instruct",
      "args": ["get fluid overload signs"]
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "ConfirmRepeatFluidBolus",
    "eventArgs": [],
    "waitFor" : {
      "name": "Recommend",
      "args": ["get fluid next step"
              , "Responsiveness to fluid therapy: true"
              , "Fluid overload: true"
              , "Recommendation: handle fluid overload"]
    }
  },
  {
    "action": "exit",
    "waitFor": {
      "name": "RecommendDrug",
      "args": [
        "Normal Saline",
        "Fluid Therapy",
        "5",
        "administer 5-10 ml/kg Normal Saline"
      ]
    }
  }
]
