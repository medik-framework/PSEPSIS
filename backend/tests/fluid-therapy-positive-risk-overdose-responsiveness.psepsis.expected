[
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get fluid overload risks"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmFluidOverloadRisksEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "RecommendDrug",
    "args": [
      "Normal Saline",
      "Fluid Therapy",
      "5",
      "administer 5-10 ml/kg Normal Saline"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get responsiveness to fluids"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get fluid overload signs"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Recommend",
    "args": [
      "get fluid next step",
      "Responsiveness to fluid therapy: true",
      "Fluid overload: true",
      "Recommendation: handle fluid overload"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "RecommendDrug",
    "args": [
      "Normal Saline",
      "Fluid Therapy",
      "5",
      "administer 5-10 ml/kg Normal Saline"
    ]
  }
]
